frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -235,7 +235,7 @@ export const EmbedFrame = ({
               </TitleAndDescriptionContainer>
             )}
             {dashboardTabs && (
-              <DashboardTabsContainer>
+              <DashboardTabsContainer narrow={!titled && pdfDownloadsEnabled}>
```

Comment:
Leaving a 60px gap on the right when there is no title (i.e. tab bar is on the same level as PDF downloads) and PDF downloads are enabled makes a lot of sense 👍🏻